### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/tutorial/sql-databases.md`

### PR DESCRIPTION
**Pull request by Zssaer**
Translates docs/zh/docs/tutorial/sql-databases.md and add deploy menu in Chinese.
related: https://github.com/tiangolo/fastapi/issues/1228

### CODE REVIEW

**Comment by Zssaer** [comment]
`jinja2.exceptions.UndefinedError: 'cookiecutter' is undefined`
is [this url](https://github.com/tiangolo/full-stack-fastapi-postgresql/tree/master/{{cookiecutter.project_slug}}/backend/app/alembic/) issue in markdown file?
how to fix this?

File: docs/zh/docs/tutorial/sql-databases.md [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+
+这将或多或少会成为一种“神奇”属性，其中表示该表与其他相关的表中的值。
+
+```Python hl_lines="2  15  26"
+{!../../../docs_src/sql_databases/sql_app/models.py!}
+```
+
+当访问 user 中的属性`items`时，如 中`my_user.items`，它将有一个`Item`SQLAlchemy 模型列表（来自`items`表），这些模型具有指向`users`表中此记录的外键。
+
+当您访问`my_user.items`时，SQLAlchemy 实际上会从`items`表中的数据库中获取项目并在此处填充进去。
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/sql-databases.md:224]:
```suggestion
当您访问`my_user.items`时，SQLAlchemy 实际上会从`items`表中的获取一批记录并在此处填充进去。
```

File: docs/zh/docs/tutorial/sql-databases.md [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+
+这意味着，例如，除非您尝试访问包含该数据的属性，否则它们不会从数据库中获取关系数据。
+
+例如，访问属性`items`：
+
+```Python
+current_user.items
+```
+
+将使 SQLAlchemy 转到`items`表并获取该用户的项目，但不是之前。
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/sql-databases.md:379]:
```suggestion
将使 SQLAlchemy 转到`items`表并获取该用户的项目，在调用`.items`之前不会去查询数据库。
```

**Review by mcthesw — APPROVED** [review]

**Review by Jax-Yang — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thanks @Zssaer! :bow: 

And thanks for the reviews @mkdir700, @Jax-Yang, @mcthesw :moon_cake:

**Pull request merged**